src/App.css

```diff
@@ -1,38 +1,144 @@
+@import url('https://fonts.googleapis.com/css2?family=Jost:ital,wght@0,100..900;1,100..900&display=swap');
+* {
+  color: white;
+}
 .App {
-  text-align: center;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  max-height: 100vh;
+  font-family: 'Jost';
+}
+
+.backdrop {
+  background-color: white;
+  display: grid;
+  justify-content: center;
+  align-items: center;
+  overflow: hidden;
+}
+
+.backdrop img, .login_form {
+  grid-column: 1;
+  grid-row: 1;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.backdrop .login_form {
+  margin-left: 255px;
+  z-index: 2;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.backdrop img {
+  transform: scale(1.5) rotate(-15deg);
+  opacity: 50%;
+  z-index: 1;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+.login_form {
```

Comment:
Overall the structure and naming conventions you used in your CSS are clear and easy to understand for me, so good job on that!